pkg/composer/exec.go

```diff
@@ -54,9 +59,29 @@ func (c *Composer) Exec(ctx context.Context, eo ExecOptions) error {
 		return fmt.Errorf("index (%d) out of range: only %d running instances from service %s",
 			eo.Index, len(containers), eo.ServiceName)
 	}
-	container := containers[eo.Index-1]
+	if len(containers) == 1 {
+		return c.exec(ctx, containers[0], eo)
+	}
+	// The sort of containers is not stable
+	// so we compare the index by its name
```

Comment:
Can't we use https://pkg.go.dev/sort#SliceStable ?

Reply:
Make sense